conda_build/os_utils/liefldd.py

```diff
@@ -1173,6 +1173,10 @@ def __call__(self, *args, **kw):
                         if not data:
                             break
                         sha1.update(data)
+                # update with file name, if its a different
+                # file with the same contents, we don't want
+                # to treat it as cached
+                sha1.update(arg.encode("utf-8"))
```

Comment:
Should we resolve potential symlinks? Those should be considered the same file indeed, right?